distage/distage-framework-docker/src/main/scala/izumi/distage/docker/examples/ZookkeeperDocker.scala

```diff
@@ -1,23 +1,29 @@
 package izumi.distage.docker.examples
 
+import java.util.UUID
+
 import distage.{ModuleDef, TagK}
-import izumi.distage.docker.ContainerDef
 import izumi.distage.docker.Docker.{ContainerConfig, ContainerNetwork, DockerPort}
+import izumi.distage.docker.{ContainerDef, DockerClientWrapper, DockerContainer}
+import izumi.distage.model.effect.{DIEffect, DIEffectAsync}
+import izumi.logstage.api.IzLogger
 
 object ZookeeperDocker extends ContainerDef {
   val zookeeperNetwork: ContainerNetwork = ContainerNetwork("zookeeper-network")
```

Comment:
We will add randomization for network names in `0.10.2` too, for now that's okay